lib/suggestion-list-element.coffee

```diff
@@ -400,13 +402,17 @@ class SuggestionListElement extends HTMLElement
   findCharacterMatchIndices: (text, replacementPrefix) ->
     return unless text?.length and replacementPrefix?.length
     matches = {}
-    wordIndex = 0
-    for ch, i in replacementPrefix
-      while wordIndex < text.length and text[wordIndex].toLowerCase() isnt ch.toLowerCase()
+    if @useAlternateScoring
+      fpm = fuzzaldrinPlus.match(text, replacementPrefix)
+      matches[pos] = true for pos in fpm
```

Comment:
yeah usually the output of fuzzaldrinPlus.match is matches. But its taken.
What about ?

````
indexes = fuzzaldrinPlus.match(text, replacementPrefix)
matches[i] = true for i in indexes 
````